source/run-command/run-command.rst

```diff
@@ -187,8 +193,148 @@ Drivers MUST document the behavior of RunCommand if a ``maxTimeMS`` field  is al
 * See Client Side Operations Timeout's section on `runCommand <https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/client-side-operations-timeout.rst#runcommand>`_
 * See Client Side Operations Timeout's section on `runCommand behavior <https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/client-side-operations-timeout.rst#runcommand-behavior>`_
 
+
+--------------------
+``runCursorCommand``
+--------------------
+
+Drivers MAY expose a runCursorCommand API with the following syntax.
+
+.. code:: typescript
+
+    interface Database {
+      /**
+       * Takes an argument representing an arbitrary BSON document and executes it against the server.
+       */
+      runCursorCommand(command: BSONDocument, options: RunCursorCommandOptions): RunCommandCursor;
+    }
+
+    interface RunCursorCommandOptions extends RunCommandOptions {
+      /**
+       * For operations that create cursors, timeoutMS can either cap the lifetime of the cursor or be applied separately to the original operation and all subsequent calls.
+       * To support both of these use cases, these operations MUST support a timeoutMode option.
+       *
+       * @defaultValue 'cursorLifetime'
+       *
+       * @see https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/client-side-operations-timeout.rst
+       */
+      timeoutMode?: 'iteration' | 'cursorLifetime';
+
+      /**
+       * Identifies the type of cursor this is.
+       *
+       * A tailable cursor can receive empty `nextBatch` arrays in `getMore` responses.
+       * However, subsequent `getMore` operations may return documents if new data has become available.
+       *
+       * A tailableAwait cursor is an enhancement where instead of dealing with empty responses the server will block until data becomes available.
+       *
+       * @defaultValue 'nonTailable'
+       */
+      cursorType: 'nonTailable' | 'tailable' | 'tailableAwait';
```

Comment:
I saw CSOT runCursorCommand tests for tailable and tailableAwait, do we need a CSOT nonTailable test?

Reply:
We had the nonTailable case, its the default so we don't need to specify it, but I've now added the two tailable cases